Runtime/Scripts/Export/ExportJobs.cs

```diff
@@ -85,6 +85,98 @@ public unsafe struct ConvertPositionFloatJob : IJobParallelFor {
             }
         }
         
+        [BurstCompile]
+        public unsafe struct ConvertMatrixJob : IJobParallelFor {
+
+            public uint byteStride;
+            
+            [ReadOnly]
+            [NativeDisableUnsafePtrRestriction]
+            public NativeArray<float4x4> input;
+            
+            [WriteOnly]
+            [NativeDisableUnsafePtrRestriction]
+            public NativeArray<float4x4> output;
+
+            public void Execute(int i) {
+                
+                var tmp = input[i];
+                tmp.c0.y *= -1;
+                tmp.c0.z *= -1;
+                
+                tmp.c1.x *= -1;
+            
+                tmp.c2.x *= -1;
+            
+                tmp.c3.x *= -1;
+
+                output[i] = tmp;
+            }
+        }
+        
+        [BurstCompile]
+        public unsafe struct ConvertSkinningJob : IJobParallelFor {
+
+            struct ushort4
+            {
+                public ushort4(uint x, uint y, uint z, uint w)
+                {
+                    this.x = (ushort)x;
+                    this.y = (ushort)y;
+                    this.z = (ushort)z;
+                    this.w = (ushort)w;
+                }
+                
+                ushort x;
+                ushort y;
+                ushort z;
+                ushort w;
+            }
+            
+            public int inputByteStride;
+            public int indicesOffset;
+            public int outputByteStride;
+            
+            [ReadOnly]
+            [NativeDisableUnsafePtrRestriction]
+            public byte* input;
+            
+            [WriteOnly]
+            [NativeDisableUnsafePtrRestriction]
+            public byte* output;
+
+            private const int unityIndicesSize = sizeof(uint) * 4;
+            private const int gltfIndicesSize = sizeof(short) * 4;
+            
+            public void Execute(int i)
+            {
+                var inputIndex = input + i * inputByteStride;
+                var outputIndex = output + i * outputByteStride;
+    
+                // if there is non-indices data before the indices data, copy it

```

Comment:
I think the Skinning indices job should not touch the other vertex attribute data. I acknowledge this is to dealt with the uint4 to ushort4 size change, but there's got to be a better way.
Doesn't this copy other vertex attributes 1:1 from the input stream and thus invalidate any conversion any other (preexisting) job did?

Reply:
Good point. Should I create another job for copying the other data over?
The weights would still be copied 1:1 though

Reply:
No. My point is those other jobs already exist! (e.g. `ConvertPositionFloatJob` used for positions and normals; your code potentially overwrites their work with incorrect data again).

What I was implying is that the size of the outputStreams should be adjusted  in first place and the byteStride will then differ between inputStream and outputStream. This has to be taken into account. It's a bit more overhead, but it's the only way to do this without doing redundant work or memory copies.

Reply:
fyi: The next release 6.5.0 will incorporate skipping unused vertex attributes, which required dealing with different input and output sizes. Consider that problem solved as a basis to evolve/rebase this PR.